mecsimcalc/quiz_utils.py

```diff
@@ -0,0 +1,113 @@
+import requests
+import jwt
+import time
+from datetime import datetime
+import smtplib
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+import logging
+
+
+# Function to append values to a Google Sheet
+def append_to_google_sheet(service_account_info, spreadsheet_id, values, range_name='Sheet1!A1', include_timestamp=True):
+    """
```

Comment:
Try using the format used for other functions in the library. It follows the NumPy docstring style guide. Your current style does not display well on MecSimCalc (not sure why)

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html